docs/docs/rules.mdx

```diff
@@ -47,6 +47,13 @@ rules:
 This example rule applies at the start of conversation as well as when the user decides
 to a send a message with an intent `greet` in the middle of an ongoing conversation.
 
+Dialogue turns that only appear as rules in the training data and do not appear in stories
+will be ignored by ML-only policies like `TEDPolicy` at prediction time.
+For example if you define the greeting rule as above and don't add it to any of your stories,
```

Comment:
Let's give actual rules / stories as example to make it less abstract for users.